scripts/generate-a3p-submission.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ueo pipefail
+
+sdkroot=$(cd -- "$(dirname "$0")/.." >/dev/null && pwd)
+
+cd $sdkroot
+
+buildSubmission() {
+  proposalName=$1
+  a3pProposal=$2
+
+  yarn agoric run packages/vats/scripts/$1.js
+
+  mkdir -p $sdkroot/a3p-integration/proposals/$2/submission
+  cp $(grep -oh '/.*b1-.*.json' $sdkroot/$1*) a3p-integration/proposals/$2/submission
+  mv $sdkroot/$1* a3p-integration/proposals/$2/submission
+}
```

Comment:
This function appears to be a manual introduction of `edit 2a7bcead2 test(a3p): restart vats # remove the restart vats part`, and AFAICT is never called by anything. Did you intend to end the file with an invocation like `buildSubmission "$@"`?

Reply:
Right it's actually a change I should land on master first. The `restart-vats` inclusion doesn't make sense in the release branch test